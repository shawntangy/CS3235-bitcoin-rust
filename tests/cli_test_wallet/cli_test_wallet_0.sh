cd ../../
cat ./tests/cli_test_wallet/cli_test_wallet_0.input | ./target/debug/bin_wallet ./bin_client/policies/seccomp_wallet.json > ./tests/cli_test_wallet/cli_test_wallet_0.output
diff ./tests/cli_test_wallet/cli_test_wallet_0.output ./tests/cli_test_wallet/cli_test_wallet_0.output.expected
