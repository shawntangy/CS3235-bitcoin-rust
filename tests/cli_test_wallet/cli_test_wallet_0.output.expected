"Initialized"
{"UserInfo":["Bob","MDgCMQDZDExOs97sRTnQLYtgFjDKpDzmO7Uo5HPP62u6MDimXBpZtGxtwa8dhJe5NBIsJjUCAwEAAQ=="]}
{"SignResponse":["[\"MDgCMQCqrJ1yIJ7cDQIdTuS+4CkKn/tQPN7bZFbbGCBhvjQxs71f6Vu+sD9eh8JGpfiZSckCAwEAAQ==\",\"MDgCMQDZDExOs97sRTnQLYtgFjDKpDzmO7Uo5HPP62u6MDimXBpZtGxtwa8dhJe5NBIsJjUCAwEAAQ==\",\"SEND $100   // By Alice   // 1678246833945\"]","ie4wMRLV+kIB/lzgOhZqG8dVz85Hzx/nuXHdvH+k2ZID0O4QXTz4b7R+Gx9ep7gO"]}
"Quitting"

